### PR TITLE
Add text-transform rule.

### PR DESCRIPTION
**Pull request by alexmingoia**

### CODE REVIEW

File: src/CSS/Text/Transform.purs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+instance valTextTransform :: Val TextTransform where
+  value t = value (show t)
+
+instance showTextTransform :: Show TextTransform where
+  show (Uppercase)  = "uppercase"
+  show (Lowercase)  = "lowercase"
+  show (Capitalize) = "capitalize"
+  show (None)       = "none"
+  show (Initial)    = "initial"
+  show (Inherit)    = "inherit"
```
Comment (garyb) [inline src/CSS/Text/Transform.purs]: This `Show` instance breaks the (informal) law that it should print the code that generates the value. Doing this for `value` would be correct, but here we'd want to print the capitalized values.

**Comment by garyb** [comment]
Sorry, I forgot there were a bunch of outstanding PRs on here. Could you sign the CLA? Then I'll get the rest of them merged/make a release. Thanks!

**Comment by alexmingoia** [comment]
CLA signed.

**Comment by garyb** [comment]
Thanks!

**Pull request merged**